endpoints/kubernetes/k8s-grpc-bookstore.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+apiVersion: v1
```

Comment:
I'd suggest adding a comment/description at the top, to explain in which circumstances this k8s descriptor should be used.
Here, I'd emphasize that this descriptor should be used on any K8s cluster that isn't running in GCP.

Reply:
I added comments to yaml files that the tutorials use.